### PR TITLE
updated-drop-down usage

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/contributing/word-list.md [hunk 1]
```diff
@@ -55,8 +55,11 @@ noun. Initial cap for “continuous” only applies when it starts a sentence.
 
 **Drill down**. Drill down, as in a menu, or other filtering feature is two words, not one.
 
-**Dropdown**. Use dropdown, not *drop-down* or *drop down*.
+**Dropdown**. Use drop-down (Adjective). For e.g., Choose your favorite model from a drop-down menu. Use dropdown (Noun). For e.g., Select a source category from the dropdown. Do not use drop down (two-words).
```
Comment (kimsauce) [inline docs/contributing/word-list.md:58]:
```suggestion
**Dropdown**.
   * Use drop-down (Adjective). Example: Choose your favorite model from a *drop-down* menu. 
   * Use dropdown (Noun). Example: Select a source category from the *dropdown*. Do not use drop down (two words).
```

**Comment by kimsauce** [comment]
I defer to Marie on this -- having two separate uses for noun/adjective will be a heavy left.

**Comment by JainM6** [comment]
> I defer to Marie on this -- having two separate uses for noun/adjective will be a heavy left.

Thanks @MariePMcGarry @kimsauce for the suggestions. I have also added a note there that says the term should not be used at all and just use UI element. In ideal world, people use dropdown when it is used as a noun( these instances are very in technical docs); and 
drop-down or any other hyphenated words are applied only when used as an adjective (often found these scenarios in technical docs).

**Comment by MariePMcGarry** [comment]
Sorry that I wasn't clear. My point was that although both `dropdown` and `drop-down` are used throughout the industry, we should settle on one usage or another. I say, since we've done a scrub to replace `dropdown` with `drop-down`, let's stick with that and move on. This was really just about consistency.

**Review by kimsauce — CHANGES_REQUESTED** [review]
see https://github.com/SumoLogic/sumologic-documentation/pull/1130#issuecomment-1343471171

**Review by stacykor — APPROVED** [review]

**Pull request merged**